src/wazuh_db/schema_upgrade_v13.sql

```diff
@@ -13,7 +13,7 @@ BEGIN;
 CREATE TABLE IF NOT EXISTS _sys_programs (
```

Comment:
Remove v13, this not is required anymore.

Reply:
Done